README.md

```
@@ -1 +1,125 @@
-# e-commerce
\ No newline at end of file
+===========================================================================================================================================================================
+============================================================================================================================================================================
+Shell script to setup the environment.
+
+#!/bin/bash
+set -e
+echo "Starting Maven project setup..."
+
+# Step 1: Install Java 11
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)